release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -825,10 +812,10 @@ application development and cluster administrator personas.
 
 Users have a choice of experience based on their role or technical abilities, including:
 
-* An admin/CaaS experience with heavy exposure to Kubernetes.
-* An AppDev/PaaS experience with standard OpenShift UX.
+* An administrator/CaaS experience with heavy exposure to Kubernetes.
```

Comment:
s/administrator/CaaS/administrator with Containers as a Service (CaaS) experience and